WordPress/Classes/ViewRelated/Pages/EditPageViewController.m

```diff
@@ -50,4 +51,14 @@ - (AbstractPost *)createNewDraftForBlog:(Blog *)blog {
     return [PostService createDraftPageInMainContextForBlog:blog];
 }
 
+#pragma mark - Onboarding
+
+- (void)showOnboardingTips
+{
+    CGFloat xValue = IS_IPAD ? CGRectGetMaxX(self.view.frame)-NavigationBarButtonRect.size.width-20.0 : CGRectGetMaxX(self.view.frame)-NavigationBarButtonRect.size.width-10.0;
```

Comment:
On readability: I find it hard to read this line as it is.  I'd suggest to use an `if` block instead, or a method if you feel it's appropriate.